library/src/webapp-filtered/editor/ckeditor.launch.js

```diff
@@ -185,6 +194,10 @@ sakai.editor.editors.ckeditor.launch = function(targetId, config, w, h) {
         filebrowserImageBrowseUrl : filebrowser.imageBrowseUrl,
         filebrowserFlashBrowseUrl : filebrowser.flashBrowseUrl,
 
+        filebrowserUploadUrl : '/direct/content/direct-upload.json?context='+siteId,
+        uploadUrl : '/direct/content/direct-upload.json?context='+siteId,
+        imageUploadUrl : '/direct/content/direct-upload.json?context='+siteId,
```

Comment:
I know they do this in other parts of this code, but I like to use this layout:
uploadUrl: `/direct/content/direct-upload.json?context=${siteId}`,

with the colon up against the key. We have these lint rules in master now, for the webcomponents stuff, and that is one of the rules. They're only warnings - it's just an attempt to make our JS more reliable. Don't go changing the entire file or anything, but I try to adjust my additions as I go.